### PR TITLE
updated function call instructions

### PR DESCRIPTION
**Pull request by TheMaja**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
My understanding of the shopping cart issue is that we are using a different property name for the shopping baskets; `shoppingBaskets` on the automarker  and `allShoppingBaskets` on the instructions. So what ultimately needs to happen is that **we should choose one name for the shopping basket property** and make sure they match by either changing the instructions or automarker config.

Changin the ordering of what parameter comes first won't make a difference since we are using object destructuring instead of positional parameters. Here's [a resource](https://blog.bitsrc.io/javascript-why-named-arguments-are-better-than-positional-arguments-9b15ab3155ef)

File: content/projects/understanding-loops/_index.md [hunk 1]
```diff
@@ -34,18 +34,18 @@ Let's say you have a function called getCustomerBaskets that takes an email addr
...
 
 // DON'T CALL YOUR FUNCTION LIKE THIS
-getCustomerBaskets("sine@umuzi.org", customerBaskets);
+getCustomerBaskets("sine@umuzi.org", shoppingBaskets);
 
 // The problem with the above code is that if you get the order of the parameters wrong, then things will break. Your functions should work like this instead:
 
 // DEFINE YOUR FUNCTION LIKE THIS
-function getCustomerBaskets({ email, allShoppingBaskets }) {...}
+function getCustomerBaskets({ shoppingBaskets, email }) {...}
```
Comment (Ngoakor12) [inline content/projects/understanding-loops/_index.md:45]: please include the actual email address above here

File: content/projects/understanding-loops/_index.md [hunk 2]
```diff
@@ -34,18 +34,18 @@ Let's say you have a function called getCustomerBaskets that takes an email addr
...
 
 // YOU CAN CALL IT LIKE THIS
-getCustomerBaskets({ email: "sine@umuzi.org", allShoppingBaskets }) //DO THIS
+getCustomerBaskets({shoppingBaskets, email}) //DO THIS
```
Comment (Ngoakor12) [inline content/projects/understanding-loops/_index.md]: same here

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
Apologies I thought I had reviewed this one. I had added comments but forgot to finalize the review

File: content/projects/understanding-loops/_index.md (hunk 1 above)
Comment (Ngoakor12) [reply content/projects/understanding-loops/_index.md:45]: missed this one

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]

File: content/projects/understanding-loops/_index.md (hunk 1 above)
Comment (Ngoakor12) [reply content/projects/understanding-loops/_index.md:45]: nevermind

**Review by Ngoakor12 — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**